easybuild/easyconfigs/s/SOCI/SOCI-4.0.3-GCC-11.2.0.eb

```diff
@@ -0,0 +1,53 @@
+easyblock = 'CMakeMake'
+
+name = 'SOCI'
+version = '4.0.3'
+
+homepage = 'http://soci.sourceforge.net/'
+description = """SOCI is a database access library for C++ that makes the illusion of embedding SQL queries in the
+ regular C++ code, staying entirely within the Standard C++."""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+
+source_urls = ['https://github.com/SOCI/soci/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['4b1ff9c8545c5d802fbe06ee6cd2886630e5c03bf740e269bb625b45cf934928']
+
+builddependencies = [
+    ('binutils', '2.37'),
```

Comment:
Unless I'm missing something: we don't need this, do we? I mean, `GCC-11.2.0` already is `GCCcore-11.2.0` + `binutils-2.37`, right?

Reply:
removing